superchain/configs/mainnet/pgn.yaml

```diff
@@ -1,4 +1,5 @@
-name: PGN
+name: PGN (Public Goods Network)
+short_name: PGN
```

Comment:
Should this be lowercase to be consistent with other chains?

Reply:
Again this is dictated by whatever is stored upstream. It can be changed upstream and then changed also here if desired.